extension/test/unit/make-payment.handler.spec.js

```diff
@@ -227,4 +229,89 @@ describe('make-payment::execute', () => {
     expect(addTransaction.transaction.state).to.equal('Failure')
     expect(addTransaction.transaction.interactionId).to.equal(JSON.parse(paymentErrorResponse).pspReference)
   })
+
```

Comment:
I guess it's better if you separate those tests into a different file.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/commit/2124321c5cdf820ced051c2882c217d9f45c1d2c